### DISCUSSION TITLE
Use Custom Font for Hubs Cloud

### DISCUSSION THREAD

**Discussion by qiangjiadong** [mozilla/hubs#3867] [Hubs Cloud Questions] [answered]
Hi Guys, I've been trying to figure out how to use our own unique font style into our Hubs Cloud on AWS.

I don't know if there is a easy way to upload our font .woff or .ttf to our Hubs Cloud AWS server and get the URL..

Or if there is another better way to customise font style for Hubs Cloud.

Thks a lot!

**Comment by robinkwilson** [comment, marked as answer]
Hello! 

A simple solution, saving you from deploying a custom client (though more "hacky"): In the HC Admin Panel > Server Settings > Advanced, you can add custom HTML and CSS code to the project to import the font then apply it to the styles. 

Otherwise you'd need to deploy a custom client with custom code. In the project you'd be import the font and apply it to the styles Hubs uses.

Hope this helps!

**Reply by qiangjiadong** [reply]
Thks Robin,

It’s super helpful!

I will have a try!!!

Thks a lot and happy Chinese New Year!!!!!

**Reply by ju1991** [reply]
Hey @robinkwilson 
is this still a possibility? In my Hubs Dashboard/Admin Panel there ist no such thing as Advanced Server Settings. I wonder if this is not possible anymore or if there is any other option to change the font of the troika text element in spoke?

**Comment by qiangjiadong** [comment]
Thks Robin,

It’s super helpful!

I will have a try!!!

Thks a lot and happy Chinese New Year!!!!!

**Reply by robinkwilson** [reply]
You're welcome! Happy Chinese new year to you too! (春节快乐!! :) )
